src/core/lib/promise/detail/basic_seq.h

```diff
@@ -404,8 +414,7 @@ class BasicSeq {
 // At each element, the accumulator A and the current value V is passed to some
 // function of type F as f(V, A); f is expected to return a promise that
 // resolves to Traits::WrappedType.
-template <template <typename Wrapped> class Traits, typename F, typename Arg,
-          typename Iter>
+template <class Traits, typename F, typename Arg, typename Iter>
```

Comment:
If we're going this far, could we also just remove the `typename F, typename Arg, typename Iter` from this type, so that we end up with `template <class Traits> class BasicSeqIter` (same for the others here)

Reply:
Done